src/main.rs

```diff
@@ -111,37 +112,37 @@ use tls::TlsOption;
 use traits::Controller;
 
 docopt!(Args derive Debug, "
-Usage: foxbox [-v] [-h] [-l <hostname>] [-p <port>] [-w <wsport>] [-d <profile_path>] [-r <url>] [-i <iface>] [-t <tunnel>] [-s <secret>] [--disable-tls] [--remote-name <hostname>] [--dns-domain <domain>] [-c <namespace;key;value>]...
+Usage: foxbox [-v] [-h] [-l <hostname>] [-p <port>] [-w <wsport>] [-d <profile_path>] [-r <url>] [-i <iface>] [-t <tunnel>] [-s <secret>] [--disable-tls] [--dns-domain <domain>] [--dns-api <url>] [--tunnel-frontend <frontend>] [-c <namespace;key;value>]...
 
 Options:
     -v, --verbose            Toggle verbose output.
-    -l, --local-name <hostname>    Set local hostname. Linux only. Requires to be a member of the netdev group.
+    -l, --local-name <hostname>    Set local hostname. [default: foxbox]
     -p, --port <port>        Set port to listen on for http connections. [default: 3000]
     -w, --wsport <wsport>    Set port to listen on for websocket. [default: 4000]
     -d, --profile <path>     Set profile path to store user data.
     -r, --register <url>     Change the url of the registration endpoint. [default: http://localhost:4242]
     -i, --iface <iface>      Specify the local IP interface.
     -t, --tunnel <tunnel>    Set the tunnel endpoint's hostname. If omitted, the tunnel is disabled.
-    -s, --tunnel-secret <secret>    Set the tunnel shared secret. [default: secret]
-        --disable-tls               Run as a plain HTTP server, disabling encryption.
-        --dns-domain <domain>       Set the top level domain for public DNS [default: box.knilxof.org]
-        --dns-api <url>             Set the DNS API endpoint [default: https://knilxof.org:5300]
-        --remote-name <hostname>    Set remote hostname. This the URL to access the box through the bridge. If omitted, the tunnel is disabled
+        --tunnel-frontend <frontend>   Set the frontend tunnel server name that clients should connect to. [default: knilxof.org]
```

Comment:
One day we might want to separate the machines external clients connect to and the machines boxes connect to.  But I think for now, it's easier to parse it out of tunnel as you suggested.